core/src/main/scala/org/locationtech/rasterframes/RasterFunctions.scala

```diff
@@ -59,6 +59,12 @@ trait RasterFunctions {
   /** Extracts the bounding box from a RasterSource or ProjectedRasterTile */
   def rf_extent(col: Column): TypedColumn[Any, Extent] = GetExtent(col)
 
+  /** Constructs a XZ2 index in WGS84 from either a Geometry, Extent, ProjectedRasterTile, or RasterSource and its CRS */
+  def rf_spatial_index(targetExtent: Column, targetCRS: Column) = XZ2Indexer(targetExtent, targetCRS)
+
+  /** Constructs a XZ2 index in WGS84 from either a ProjectedRasterTile or RasterSource */
```

Comment:
ah i see. the _index space_ is in WGS84 but the CRS may not be ...  This is interesting. I had read too fast and assumed if no CRS given, it would assume WGS84. But that is not the case.